install/kubernetes/helm/istio/charts/tracing/values.yaml

```diff
@@ -29,9 +29,60 @@ podAntiAffinityTermLabelSelector: {}
 
 jaeger:
   hub: docker.io/jaegertracing
-  tag: 1.9
+  tag: 1.11
   memory:
     max_traces: 50000
+  operator:
+    enabled: false
+  # The spec represents the Jaeger operator's custom resource (CR).
+  # CR docs can be found here: https://github.com/jaegertracing/jaeger-operator#creating-a-new-jaeger-instance
+  # Some example CRs: https://github.com/jaegertracing/jaeger-operator/tree/master/deploy/examples
+  spec:
```

Comment:
Not sure if this is a good idea to put the spec here in `values.yaml`

Reply:
Main reason was that, if we provide the fully expanded CR in the values.yaml using default values, then the user could override any aspect via the helm cli.

However open to moving the CR details into a separate yaml with restricted range of properties in the values file if that is the preferred way.

@douglas-reid thoughts?

Reply:
@objectiser I would say move the CR details into a separate yaml with restricted range of properties , as you suggested , that way we have a cleaner values file.